### ISSUE TITLE
[Forge 1.19.2, ModernFix 5.18.0] Creative search is empty

### ISSUE THREAD

**Issue by katubug** [embeddedt/ModernFix#465]
### Bug Description

I saw this was in several closed issues, but I am having it occur on version 5.18.0 for 1.19.2.

Here is what appears several times in my latest.log - 
```
[01Oct2024 16:24:30.091] [Render thread/ERROR] [ModernFix/]: Error searching
java.lang.invoke.WrongMethodTypeException: expected (IngredientFilter,String)Stream but found (IngredientFilter,String)List
	at java.lang.invoke.Invokers.newWrongMethodTypeException(Invokers.java:523) ~[?:?]
	at java.lang.invoke.Invokers.checkExactType(Invokers.java:532) ~[?:?]
	at org.embeddedt.modernfix.searchtree.JEIBackedSearchTree.searchJEI(JEIBackedSearchTree.java:78) ~[modernfix-forge-5.18.0+mc1.19.2.jar%23750!/:5.18.0+mc1.19.2]
	at org.embeddedt.modernfix.searchtree.JEIBackedSearchTree.m_6293_(JEIBackedSearchTree.java:65) ~[modernfix-forge-5.18.0+mc1.19.2.jar%23750!/:5.18.0+mc1.19.2]
	at net.minecraft.client.gui.screens.inventory.CreativeModeInventoryScreen.m_98630_(CreativeModeInventoryScreen.java:379) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.gui.screens.inventory.CreativeModeInventoryScreen.m_5534_(CreativeModeInventoryScreen.java:289) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.KeyboardHandler.lambda$charTyped$5(KeyboardHandler.java:418) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:522) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.KeyboardHandler.m_90889_(KeyboardHandler.java:416) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.KeyboardHandler.m_167827_(KeyboardHandler.java:446) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:103) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.KeyboardHandler.m_90934_(KeyboardHandler.java:445) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at org.lwjgl.glfw.GLFWCharModsCallbackI.callback(GLFWCharModsCallbackI.java:46) ~[lwjgl-glfw-3.3.1.jar%2384!/:build 7]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.1.jar%2396!/:build 7]
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474) ~[lwjgl-glfw-3.3.1.jar%2384!/:build 7]
	at com.mojang.blaze3d.systems.RenderSystem.m_69830_(RenderSystem.java:222) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1146) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:700) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.main.Main.m_239872_(Main.java:212) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[client-1.19.2-20220805.130853-srg.jar%23869!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.2-43.4.4.jar%23150!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.8.jar%23137!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) [bootstraplauncher-1.1.2.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:67) [ForgeWrapper-prism-2024-02-29.jar:prism-2024-02-29]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
```

https://i.imgur.com/NH8DKMR.png

Upon further study, I've found that disabling the blast_search_trees fixes the issue, but since it was reportedly fixed in recent versions, I figured I'd report that it's still broken in certain circumstances.

### Reproduction Steps

1. Using Cottage Witch modpack, in dev version
2. Using ModernFix 1.18.0, load game
3. Attempt to search via the Creative search bar

### Log File

https://gist.github.com/katubug/d511d5e92b92b7378aff5894e4cfb530

**Comment by embeddedt** [comment]
The issue is fixed on versions of ModernFix for 1.20.1 and newer. I didn't expect JEI to backport those changes to 1.19. I will have to release another 1.19 version at some point when I have time. For now, the only solution is to disable `blast_search_trees` or downgrade JEI.

**Comment by muggleorwizard** [comment]
Same with 1.18.2 :/

**Comment by embeddedt** [comment]
5.18.1 has been released for 1.19.2, fixing this.
